app-config.yaml

```diff
@@ -21,6 +21,8 @@ app:
         links:
           - url: https://discord.gg/MUpMjP2
             title: '#backstage'
+  catalog:
+    apiMenu: true
```

Comment:
As mentioned in the discussions around this feature request, I think it'd be unfortunate to design things this way. The config approach would be very coarse and probably not what we'd want longer term.

What we want to do is instead to turn the control a bit inside out, so that users compose the pages the way that they see fit. You'd be in control of the "page chrome" at the JSX level and would just exclude this part as you see fit.

Sorry but I don't see a short term good fix that solves this well for your ask. Maybe somebody else could pitch in with ideas, but I feel that this particular one, isn't it.

Reply:
Hi @freben .

I understand you point of view and I would like to fine another solution, Just for my understanding are you suggest to use Entity configuration?, like add that config in the api file and extract using this function https://github.com/backstage/backstage/blob/master/plugins/catalog/src/components/EntityLayout/EntityLayout.tsx#L114 ? 
Or is there another solution in mind ?